donut/get-commits.js

```diff
@@ -0,0 +1,31 @@
+export default function getCommits(){
+
+	var c1 = fetch('https://api.github.com/repos/Financial-Times/polyfill-service/commits?access_token=6383432658b8e7a508f0eb1d78b0a9efec4b5d97&page=1&per_page=50')
+	.then(function(repo){
+		return repo.json();
+	})
+	.catch(function(error){
+		console.log(error);
+	});
+	var c2 = fetch('https://api.github.com/repos/Financial-Times/polyfill-service/commits?access_token=6383432658b8e7a508f0eb1d78b0a9efec4b5d97&page=2&per_page=50')
+	.then(function(repo){
+		return repo.json();
+	})
+	.catch(function(error){
```

Comment:
`.catch` doesn't need to happen on every request. Could be added only after the `Promise.all`